### PR TITLE
Fix SDL OnSeekMediaClockTimer notification from HMI

### PR DESCRIPTION
**Pull request by indlu**
Fixed OnSeekMediaClockTimer notification from HMI
 Added to MessageBroker subscription on missing notification UI.OnSeekMediaClockTimer

### CODE REVIEW

**Comment by indlu** [comment]
Please review @LevchenkoS, @dcherniev, @okozlovlux , @LuxoftAKutsan

File: src/components/application_manager/src/commands/hmi/on_seek_media_clock_timer_notification.cc [hunk 1]
```diff
@@ -53,7 +53,15 @@ void OnSeekMediaClockTimerNotification::Run() {
   (*message_)[strings::params][strings::function_id] =
       static_cast<int32_t>(mobile_apis::FunctionID::OnSeekMediaClockTimerID);
 
-  SendNotificationToMobile(message_);
+  const ApplicationSet& accessor =
+      application_manager_.applications().GetData();
+
+  ApplicationSetIt it = accessor.begin();
+  for (; accessor.end() != it;) {
+    const ApplicationSharedPtr app = *it++;
```
Comment (LevchenkoS) [inline src/components/application_manager/src/commands/hmi/on_seek_media_clock_timer_notification.cc]: @indlu, this expression does not look so obvious.
Please, move increment operation to head of `for` loop.

Comment (indlu) [inline src/components/application_manager/src/commands/hmi/on_seek_media_clock_timer_notification.cc]: @LevchenkoS, fixed in 92b13ff

File: src/components/application_manager/test/commands/hmi/on_seek_media_clock_timer_notification_test.cc [hunk 2]
```diff
@@ -51,20 +52,67 @@ using ::utils::SharedPtr;
...
+  }
+
+ protected:
+  am::ApplicationSet application_set_;
+  sync_primitives::Lock applications_lock_;
+  DataAccessor<am::ApplicationSet> applications_;
+  am::ApplicationSharedPtr app_;
+  NiceMock<MockApplication>* app_ptr_;
+
+  am::ApplicationSharedPtr ConfigureApp(NiceMock<MockApplication>** app_mock,
```
Comment (dcherniev) [inline src/components/application_manager/test/commands/hmi/on_seek_media_clock_timer_notification_test.cc]: @indlu Please note that it is preffered to name mock objects with "mock_" prefix so please rename the app_mock variable into mock_app, and I also think you need to add mock_ prefix to app_ptr_.

File: src/components/application_manager/src/commands/hmi/on_seek_media_clock_timer_notification.cc (hunk 1 above)
Comment (indlu) [inline src/components/application_manager/src/commands/hmi/on_seek_media_clock_timer_notification.cc]: @LevchenkoS, fixed in 5220616

Comment (indlu) [inline src/components/application_manager/src/commands/hmi/on_seek_media_clock_timer_notification.cc]: @LevchenkoS, fixed in 5220616

File: src/components/application_manager/test/commands/hmi/on_seek_media_clock_timer_notification_test.cc (hunk 2 above)
Comment (indlu) [inline src/components/application_manager/test/commands/hmi/on_seek_media_clock_timer_notification_test.cc]: @LevchenkoS, fixed in 4da9893

**Pull request merged**